### ISSUE TITLE
deleting a character or inserted text is not read by screen readers in terminal

### ISSUE THREAD

**Issue by hamidrezaabroshan** [microsoft/vscode#212025]
Does this issue occur when all extensions are disabled?: Yes
• VS Code Version: 1.88.1
• OS Version: Windows_NT x64 10.0.22631
Steps to Reproduce:
1. Turn on NVDA screen reader.
2. Open VS Code and press Ctrl+` to open the terminal.
3. Type something in the terminal. If you delete a character or use Insert+Up Arrow, the screen reader does not announce the changes.
4. This issue does not occur in PowerShell or CMD.
Expected: Screen readers should announce all text changes in the terminal.
Actual: No announcements when text is deleted or read using Insert+Up Arrow.

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! It looks like you may be using an old version of VS Code, the latest stable release is 1.89.0. Please try upgrading to the latest version and checking whether this issue remains.

Happy Coding!

**Comment by meganrogge** [comment]
This works for me on macOS with Voiceover. We rely on shell integration for text area syncing. My guess is shell integration is not be working for you. Could you pls:

1) Run the command: `Developer: Set Log Level... `
2) Select `Terminal`
3) Select `Trace`
4) create a terminal and type a simple command, then backspace or up arrow
5) Run the command: `Output: Show Output Channels...`, select `Terminal`
6) Copy and paste the output to this issue

**Comment by meganrogge** [comment]
probably a duplicate of https://github.com/microsoft/vscode/issues/233921
